### DISCUSSION TITLE
Shiny Jobs not working on Android - This.ShinyOnCreate() not available

### DISCUSSION THREAD

**Discussion by StantonR-ZA** [shinyorg/shiny#1084] [Q&A] [answered]
Hi There

I am using Shiny Jobs and I am having some issues.

I have followed the samples but I cant get Shiny jobs to work on android.

This.ShinyOnCreate() is not available in my android project.

I have installed all nuget packages in my android project.
I am using vs2019 community edition
I am targeting version 10.

Is there anything else I am missing. I have already spent so many hours on this.
Is there any other info required in order to help me resolve this.

![image](https://user-images.githubusercontent.com/111248412/187250318-a1c55372-d4f7-4e58-987b-b7bc21f232e2.png)
![image](https://user-images.githubusercontent.com/111248412/187250542-7362a556-b1cf-4a22-9b74-1ff92186bda9.png)
![image](https://user-images.githubusercontent.com/111248412/187250638-e2fbc3f4-a025-4b9c-b269-410ff78d8084.png)

**Comment by aritchie** [comment, marked as answer]
https://github.com/shinyorg/shiny/issues/1059

**Reply by StantonR-ZA** [reply]
Thanks so much. Should I downgrade the Shiny version then.

**Reply by aritchie** [reply]
That's a decision you need to make.  I don't take issues for old versions of Shiny.  

Judging by your package refs, you have a lot of old android support libraries that are going to be in the way of androidx updates.
